Rise-Vision/rise-vision-apps
# Open RSS component settings [stage-8]

## Description
Opens the ( empty ) RSS settings editor and adds the streamline RSS icon 

## Motivation and Context
RSS component UI Settings development

## How Has This Been Tested?
Tested locally and on staging

https://apps-stage-8.risevision.com/templates/edit/82f2346a-7548-485e-bcf6-2d8c38809af9/?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
  - Manually tested
  - Initial automated unit tests added
  - Deployment will occur on Monday. Will monitor by testing production after released. There are no templates visible to users that use _rise-data-rss_ component in them, very low risk deployment
  - Rollback will involve re-running last master CCI build followed by reverting code changes in repo
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
- No documentation required
- Support not required
